.circleci/config.yml

```diff
@@ -270,8 +271,19 @@ jobs:
               . activate habitat; cd habitat-lab
               export PYTHONPATH=.:$PYTHONPATH
               export MULTI_PROC_OFFSET=0 && export MAGNUM_LOG=quiet && export HABITAT_SIM_LOG=quiet
-              python -m pytest --cov-report=xml --cov-report term  --cov=./
+              pytest test/ --cov-report=xml --cov-report term  --cov=./
```

Comment:
Any story here? Was calling pytest from module failing?

Reply:
This was not failing - just wanted to standardize test calls. I'll use the other form to avoid unnecessary changes.